deploy/v1.0.0/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
pls only make change to `deploy` folder, `v1.0.0` is already released, should not change.

Reply:
@andyzhangx done :)